ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md

```diff
@@ -967,6 +967,8 @@ wan_virtual_topologies:
 | AvdTestBGP | VerifyBGPSpecificPeers | Validate the state of BGP Address Family sessions, including `Path-Selection` for AutoVPN, `Link-State` and `IPv4/IPv6 SR-TE` for CV Pathfinder. |
 | AvdTestIPSecurity | VerifySpecificIPSecConn | Validate the establishment of IP security connections for each static peer under the `router path-selection` section of the configuration. |
 | AvdTestStun | VerifyStunClient | Validate the presence of a STUN client translation for a given source IPv4 address and port. The list of expected translations for each device is built by searching local interfaces in each path-group. |
+| AvdTestAvtPath | VerifyAVTSpecificPath | Validate that the status is active and the type is direct for an Adaptive Virtual Topology (AVT) path in a specified VRF. |
```

Comment:
need to explain that this is only checking the static peers (for now, lets see if we can be smarter later)

Reply:
change also the PR description and title to indicate this

Reply:
updated